File: python/how-many-seconds-have-passed.py
import os
import time
i = 1
try:
	while True:
		print("How many seconds have passed since you started this program?\n" + str(i) + "\n\nPress Ctrl+C to exit")
		time.sleep(1)
		os.system("clear")
		i += 1
except KeyboardInterrupt:
	pass

File: javascript/how-many-seconds-have-passed.js
let seconds = 0;
const s = "How many seconds have passed since you started this program?\n";
function count() {
	seconds += 1;
	console.log(s + seconds + "\n\nPress Ctrl+C to exit");
	function c() {
		console.clear();
		}
	setInterval(c,1000);
	}
setInterval(count,1000);
